src/config/docker.md

```diff
@@ -0,0 +1,96 @@
+# Docker
+
+Docker is a platform of products to build isolated containers for programs using
+visualization.
```

Comment:
visualization? Either I misunderstood this sentence, which means it should be probably worded differently, or `visualization` should be replaced with containarization  or virtualization?

Reply:
It should be virtualization

Reply:
namespaces have nothing to with virtualization.